src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -37,6 +37,9 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="$(MSBuildThisFileDirectory)Internal\AssemblyAttributes.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Internal\Console.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Internal\Console.Unix.cs" Condition="'$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true'" />
+    <Compile Include="$(MSBuildThisFileDirectory)Internal\Console.Windows.cs" Condition="'$(TargetsWindows)' == 'true'" />
```

Comment:
There are large platform specific sections bellow, it might be better to follow the existing pattern and add this to e.g. https://github.com/MichalStrehovsky/runtime/blob/commonConsole/src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems#L1226